packages/chart/src/ChartUtils.js

```diff
@@ -1206,39 +1217,49 @@ class ChartUtils {
       });
     } else if (type === dh.plot.AxisType.Y) {
       Object.assign(axis, {
-        zerolinecolor: ChartTheme.zerolinecolor,
+        zerolinecolor: theme.zerolinecolor,
         zerolinewidth: 2,
       });
     }
 
     return axis;
   }
 
-  static makeDefaultLayout() {
+  static makeDefaultLayout(theme = {}) {
+    let colorway = [];
+    if (theme.colorway) {
```

Comment:
I'd put this in another static method, `getColorWayFromThemeProperty` or something. Just to keep this method clean